### PR TITLE
Reproducibility improvements

### PR DESCRIPTION
**Pull request by adswa**
A few improvements to allow generating the manuscript without computations (by including the figures), and to improve the reproducibility of the figures, by removing time stamps and making SVG identifiers deterministic.
If you could sanity check this, @mih, I'd be very grateful

### CODE REVIEW

**Comment by adswa** [comment]
Do you have any objections to specifying precise software versions of the python packages, @mih? I may add this to this PR

File: code/mk_figuresnstats.py [hunk 1]
```diff
@@ -30,6 +30,9 @@
 )
 from remodnav.tests import utils as ut
 
+# configure matplotlib for deterministically named SVG identifiers.
+from matplotlib import rcParams
+rcParams['svg.hashsalt'] = 43
```
Comment (mih) [inline code/mk_figuresnstats.py:35]: Wouldn't it be 42? ;-)

**Review by mih — APPROVED** [review]
This makes a lot of sense! Thx!

File: code/mk_figuresnstats.py (hunk 1 above)
Comment (adswa) [reply code/mk_figuresnstats.py:35]: I wanted to surprise people ;-)

**Comment by mih** [comment]
> Do you have any objections to specifying precise software versions of the python packages, @mih? I may add this to this PR

In the virtual env setup to attempt to recreate the one we had around the time of submission? Why not, indeed!

**Comment by adswa** [comment]
Okay, I made some slight changes and force-pushed

**Comment by adswa** [comment]
I'll merge this now

**Pull request merged**